File: Bekend/AirlineApi/Models/Airline.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace AirlineApi.Models
{
    public class Airline
    {
        [Key]
        public int Id { get; set; }
        public string Description { get; set; }
        public string CompanyName { get; set; }
        public string Adress { get; set; }

        public int Mark { get; set; }

        public ICollection<Flight> Flights { get; set; }

        
    }
}

File: projekat/src/app/entities/airline/airline.ts
import { Flight } from '../flight/flight';

export class Airline{

    id: number;
    name: string;
    address: string;
    description: string;
    mark: number;
    activated:boolean;
    flights= new Array<Flight>();

    adminName= "";
  
    

    constructor(id:number,name: string,address: string,description: string,mark: number,adminName :string)
    {
        this.id = id;
        this.name = name;
        this.address = address;
        this.description = description;
        this.mark = mark;
        this.adminName=adminName;
    }
}